pkg/rslog/debug.go

```diff
@@ -6,106 +6,125 @@ import (
 	"sync"
 )
 
+// ProductRegion is a numerical value assigned to an area of the product.
 type ProductRegion int
 
-// regionNames translates the enum region to its string equivalent. This is
-// used for display (when logging) and also when processing the configuration
-// values.
-var regionNames map[ProductRegion]string
+// debugMutex protects access to product debug region/enabled information.
+var debugMutex sync.RWMutex
```

Comment:
Renamed the globals to clearly indicate that they are only for debug log handling.